### DISCUSSION TITLE
Support for Scala 2.12.14?

### DISCUSSION THREAD

**Discussion by ambantis** [scalameta/metals#2899] [General]
I got a message that metals does not yet support Scala 2.12.14. I'm working in a monorepo with ScalaFix; we're using 2.12.14 and 2.13.6. Downgrading 2.12.14 is not an option.

Is there an ETA for support; is there an issue I can watch so that I can keep tabs on this?

I'm currently using 0.10.4+93-1da3997b-SNAPSHOT

**Comment by gabro** [comment]
Support for 2.12.14 was added in Metals 0.10.4 stable. See https://scalameta.org/metals/blog/2021/05/31/tungsten.html

You just need to update to that version (vim and vscode should prompt you about this automatically)

**Comment by tgodzik** [comment]
You might also have a different Metals version set in a local workspace settings if you are using VS Code, which is something that happened to me a couple of times.
